jenkins-formula/tests/test_02_jenkins_agent.py

```diff
@@ -0,0 +1,29 @@
+"""
+Test suite for assessing the Jenkins Agent configuration results
+Copyright (C) 2023 Georg Pfuetzenreuter <georg.pfuetzenreuter@suse.com>
+
+This program is free software: you can jenkinstribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+sysconffile = f'/etc/sysconfig/jenkins-agent'
```

Comment:
Just wondering - why is this a `f` string?

Reply:
I think it's a leftover from copying some existing test as a template. Thanks, removed.